### PR TITLE
[HOTFIX]: Pass SparkSession to LoadEvents

### PR DESCRIPTION
**Pull request by vikramahuja1001**
### Why is this PR needed?
 Pass SparkSession to LoadEvents
 
 ### What changes were proposed in this PR?
 Pass SparkSession to LoadEvents
 
### Does this PR introduce any user interface change?
 - No
 
 ### Is any new testcase added?
 - No

### CODE REVIEW

File: processing/src/main/java/org/apache/carbondata/processing/loading/events/LoadEvents.java [hunk 1]
```diff
@@ -58,6 +62,10 @@ public LoadTablePreExecutionEvent(CarbonTableIdentifier carbonTableIdentifier,
       this.carbonLoadModel = carbonLoadModel;
     }
 
+    public SparkSession getSparkSession() {
```
Comment (Indhumathi27) [inline processing/src/main/java/org/apache/carbondata/processing/loading/events/LoadEvents.java:65]: why it is required?

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/1041/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2754/

File: processing/pom.xml [hunk 2]
```diff
@@ -40,6 +40,11 @@
       <artifactId>carbondata-core</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-sql_${scala.binary.version}</artifactId>
```
Comment (jackylk) [inline processing/pom.xml:45]: should not depends on spark in core module (carbon-core, carbon-processing)

**Pull request closed without merging**